fbw-a32nx/src/systems/instruments/src/FCU/FCU.tsx

```diff
@@ -9,30 +9,42 @@ import {
 } from '@microsoft/msfs-sdk';
 import { EisDisplay } from 'instruments/src/FCU/Components/EisDisplay';
 import { AfsDisplay } from 'instruments/src/FCU/Components/AfsDisplay';
+import { FcuSimvars } from 'instruments/src/FCU/shared/FcuSimvarPublisher';
+import { A32NXElectricalSystemEvents } from '../../../shared/src/publishers/A32NXElectricalSystemPublisher';
 
 import './style.scss';
-import { FcuSimvars } from 'instruments/src/FCU/shared/FcuSimvarPublisher';
 
 interface FCUProps extends ComponentProps {
   bus: EventBus;
 }
 
 export class FCUComponent extends DisplayComponent<FCUProps> {
-  private fcuHealthy = ConsumerSubject.create(null, false);
+  private readonly sub = this.props.bus.getSubscriber<A32NXElectricalSystemEvents & FcuSimvars>();
 
-  onAfterRender(node: VNode): void {
-    super.onAfterRender(node);
+  private readonly fcuHealthy = ConsumerSubject.create(this.sub.on('fcuHealthy'), false);
 
-    const sub = this.props.bus.getSubscriber<FcuSimvars>();
-
-    this.fcuHealthy.setConsumer(sub.on('fcuHealthy'));
-  }
+  private readonly isBacklightPowered = ConsumerSubject.create(this.sub.on('a32nx_elec_ac_1_bus_is_powered'), false);
+  private readonly isLcdPowered = ConsumerSubject.create(this.sub.on('a32nx_elec_dc_1_bus_is_powered'), false);
```

Comment:
Checking if the LCD is powered is redundant, since I'm pretty sure it is contained in the fcuHealthy simvar (if the FCU is not powered, it is not healthy).
In this case it is even incorrect, as the FCU channel 1 is powered by DC ESS, and channel 2 by DC 2

Reply:
Totally missed that's already there. Easy to move it out so we avoid mutating inside the SVG and save at least one total redraw of the FCU. We should rewrite the display in HTML at some point to gain a bit of perf.